lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -2905,8 +2905,7 @@ std::optional<SwiftNominalType> GetSwiftClass(ValueObject &valobj,
 
   auto isa_load_addr = descriptor_sp->GetISA();
   Address isa;
-  const auto &sections = objc_runtime.GetTargetRef().GetSectionLoadList();
-  if (!sections.ResolveLoadAddress(isa_load_addr, isa))
+  if (!objc_runtime.GetTargetRef().ResolveLoadAddress(isa_load_addr, isa))
```

Comment:
Is this forwarding the call to the SectionLoadList?

Reply:
[Yes](https://github.com/swiftlang/llvm-project/commit/c4fb7180cbbe977f1ab1ce945a691550f8fdd1fb#diff-78749e4b6882532f7152ce60a7923c835ec0fc2aec8bb07d19e8bcdfb1198860).